backend/core/database.py

```diff
@@ -1362,11 +1372,20 @@ def _update_bgp_updates(self):
                         psycopg2.extras.execute_values(
                             db_cur, query, list(update_bgp_entries), page_size=1000
                         )
-                    query = "UPDATE bgp_updates SET handled=true, hijack_key=array_distinct(hijack_key || array[data.v1]) FROM (VALUES %s) AS data (v1, v2, v3) WHERE bgp_updates.key=data.v2"
+                    update_bgp_entries_no_time_thres = set()
+                    for update_bgp_entry in update_bgp_entries:
+                        update_bgp_entries_no_time_thres.add(
+                            (update_bgp_entry[0], update_bgp_entry[1])
+                        )
+                    query = "UPDATE bgp_updates SET handled=true, hijack_key=array_distinct(hijack_key || array[%s]) WHERE bgp_updates.key=%s"
```

Comment:
Here I also refactored the syntax to be compatible with execute_batch instead of values since values runs everything in parallel; good for performance, bad for correctness since there can be race conditions when updating the hijack keys for the same hijack (the most recent wins, which is erroneous).

Reply:
Update: I kept execute values for withdrawals that do not have this dependency!